pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/CastTransformFunction.java

```diff
@@ -84,6 +84,22 @@ public void init(List<TransformFunction> arguments, Map<String, DataSource> data
         case "JSON":
           _resultMetadata = JSON_SV_NO_DICTIONARY_METADATA;
           break;
+        case "INTEGERMV":
+        case "INTMV":
```

Comment:
We should probably not need to introduce this. Won't look clean in the SQL query.

Reply:
We should be able to detect that column being casted is SV or MV. `INTMV` is no type and looks like a hack

Reply:
Done.